Packs/IBMResilientSystems/TestPlaybooks/playbook-IBM_Resilient_Systems_Test.yml

```diff
@@ -324,19 +393,139 @@ tasks:
       iscommand: false
       brand: ""
     separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 2295
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: c7d7f6f0-89e5-49b1-8cf6-6e0e4217515e
+    type: regular
+    task:
+      id: c7d7f6f0-89e5-49b1-8cf6-6e0e4217515e
+      version: -1
+      name: rs-add-notes
+      script: IBM Resilient Systems|||rs-add-notes
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "12"
+    scriptarguments:
+      comment:
+        simple: This is a note
+      incident-id:
+        simple: ${Resilient.Incidents.Id}
+    separatecontext: false
     view: |-
       {
         "position": {
           "x": 50,
           "y": 1770
         }
       }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: bc055c52-c7bb-4302-8bb0-7ced5ce44abf
+    type: regular
+    task:
+      id: bc055c52-c7bb-4302-8bb0-7ced5ce44abf
+      version: -1
+      name: rs-add-artifact
+      script: IBM Resilient Systems|||rs-add-artifact
+      type: regular
+      iscommand: true
+      brand: IBM Resilient Systems
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      artifact-description:
+        simple: This is a description
+      artifact-type:
+        simple: IP Address
+      artifact-value:
+        simple: 1.1.1.1
+      incident-id:
+        simple: ${Resilient.Incidents.Id}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1945
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: ab83891c-6cc5-416b-86fb-2d23d007c2c9
+    type: regular
+    task:
+      id: ab83891c-6cc5-416b-86fb-2d23d007c2c9
+      version: -1
+      name: Update incident using other-fields argument
+      description: Updates incidents using other-fields argument.
+      script: IBM Resilient Systems|||rs-update-incident
+      type: regular
+      iscommand: true
+      brand: IBM Resilient Systems
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      incident-id:
+        complex:
+          root: Resilient.Incidents
+          accessor: Id
+      other-fields:
+        simple: '{"description": {"textarea": {"format": "html", "content": "The new
+          description"}}, "name": {"text": "The new name"}}'
+    separatecontext: false
```

Comment:
Where are we testing that the field was updated successfully?

Reply:
I do not know how to check it because we add every day (when the TPB is running) a new note and we do not remove it after that (there is no command that removes a note). So if we check that the note exists it could be a note from yesterday.

Reply:
Can we create a new incident in the beginning of the test and delete it in the end?